dsgrid/dataset/dataset_schema_handler_standard.py

```diff
@@ -40,10 +51,123 @@ def get_unique_dimension_rows(self):
         """Get distinct combinations of remapped dimensions, including id.
         Check each col in combination for null value."""
         dim_table = self._remap_dimension_columns(self._load_data_lookup).distinct()
-        self._check_null_value_in_unique_dimension_rows(dim_table)
-
+        check_null_value_in_unique_dimension_rows(dim_table)
         return dim_table
 
+    def make_project_dataframe(
+        self, load_data_df=None, lookup_df=None, query_context: QueryContext = None
+    ):
+        ld_df = load_data_df or self._load_data
+        lk_df = lookup_df or self._load_data_lookup
+
+        if query_context is not None:
+            lk_df, ld_df = self._prefilter_dataset(query_context, lk_df, ld_df)
+
+        lk_df = self._remap_dimension_columns(lk_df)
+        ld_df = self._remap_dimension_columns(
+            ld_df,
+            # Some pivot columns may have been removed.
+            pivot_columns=set(ld_df.columns).intersection(self.get_pivot_dimension_columns()),
+        )
+
+        if query_context is not None:
+            ld_df = self._process_metric_aggregations(query_context, ld_df)
+
+        ld_df = ld_df.join(lk_df.drop("fraction"), on="id").drop("id")
```

Comment:
Did "fraction" get worked into the load_data table at this point? If not, we cannot drop col.

Reply:
That should have happened in `_remap_dimension_columns` and/or `_process_metric_aggregations`, right? Perhaps it should be dropped as soon as it is applied.

Reply:
In the standard dataset schema, the fraction col often only exists in the lookup table, although I suppose the load_data table could have one too if a mapping with scaling factors is applied to the time column. When you join the two tables in the standard schema, the fraction(s) need to be applied to the value columns and be consolidated that way. Though we can leave fraction as is and apply during the final aggregation step.

Reply:
Deferred from this PR.